engine/battle/effect_commands.asm

```diff
@@ -1902,6 +1902,7 @@ BattleCommand_checkhit:
 	ret
 
 .NoGuardCheck:
+;Returns z if either the user or opponent has the No Guard ability
```

Comment:
Nitpicking: Comments always have a space after the `;`. Same in `.AntiMinimze`.

Reply:
oops :V